log_transfer/tests/audit_logging.py

```diff
@@ -34,38 +37,56 @@ def log(
     get_time: Callable[[], datetime] = _now,
     ip_address: str = "",
     additional_information: str = "",
+    use_django_auditlog: bool = False,
```

Comment:
Rather than having a boolean for the type of logger, setting this as enum or string with default value would be better for possible future logging types. -> change to something like "auditlogger_type: str = "SINGLE_COLUMN_JSON", where the other possible value would be "DJANGO_AUDITLOG". This requires some changes throughout the code though.

Reply:
Yes, good suggestion. I think this should also apply to the "USE_DJANGO_AUDITLOG" environment variable, which should then be an enum. Do you prefer integer or string enums?

Reply:
For ENV variables that need to be set up externally I would prefer using String values, as the meaning of the value will be clearer when someone reads the env configuration, even though it is easier to get wrong initially, ENV input validation and fail fast with error printed also to stdout might work for potential issues, the config error will be noticed and visible quicker when running in, for example, openshift. 
For internal variables I don't have preference, you probably know better how things are commonly done in python, you can go either way with those.

Reply:
I added this change. I initially had `AUDIT_LOGGER_TYPE` default to `SINGLE_COLUMN_JSON` if an unexpected value was defined with a warning given, but I think it makes more sense to fail fast. The error given for incorrect env value is now: `ValueError: 'foo' is not a valid AuditLoggerType`. Checking the mentioned enum should be simple enough when configuring the application.